frontend/src/app/settings/modelsources/t4c-settings/edit-t4c-instance/edit-t4c-instance.component.html

```diff
@@ -80,15 +105,18 @@ <h2 *ngIf="existing"></h2>
             autocomplete="new-password"
             formControlName="password"
           />
+          <mat-error *ngIf="form.controls.password.errors?.required">
+            The password is required.
+          </mat-error>
         </mat-form-field>
       </fieldset>
-      <div *ngIf="existing == 'editing'" id="editing-button-wrapper">
+      <div *ngIf="editing" id="editing-button-wrapper">
         <button mat-flat-button color="warn" (click)="cancelEditing()">
           Cancel
         </button>
         <button mat-flat-button type="submit" color="primary">Submit</button>
       </div>
-      <div *ngIf="existing == 'existing'">
+      <div *ngIf="!editing">
```

Comment:
This condition should be `!editing && existing`, else the button appears on creation step.